maturity.go

```diff
@@ -25,6 +27,21 @@ func (s *MaturityReport) Get(category string) *Level {
 	return nil
 }
 
+func (c *Client) GetServiceMaturityWithAlias(alias string) (*ServiceMaturity, error) {
```

Comment:
can we support ID or Alias with this call?  Using that NewIdentifier stuff that detects a GID from an Alias?

If not thats fine

Reply:
Let me try that.  I think that should work.

Reply:
Is there something that has to be implemented on the API side to get this to work?  I tried passing in an IdentifierInput param on the GQL side and it's saying that it's not an accepted type.  I don't remember where else we did this for a query, only seeing mutation examples.

Reply:
Looking at the graphql API docs for this it doesn't support the `IdentifierInput` yet only `id` or `alias` because each is nullable.  Here is the signature of the query:

```
service(id: ID alias: String): Service
```

Which means you'd need to pass null for `id` or `alias` depending on which value is given by the enduser using `GetServiceMaturityWithAlias`

Something like
```
	var q struct {
		Account struct {
			Service ServiceMaturity `graphql:"service(id: $id alias:$alias)"`
		}
	}
	v := PayloadVariables{
            "id": nil,
            "alias": nil,
        }
	if IsID(input) {
	    v["id"] = graphql.ID(input)
	} else {
	    v["id"] = graphql.String(input)
	}
	
```